### DISCUSSION TITLE
Detect "throttled" state?

### DISCUSSION THREAD

**Discussion by dwm66** [nobl/ioBroker.senec#506] [General]
Is there a datapoint, which can be used to detect if the system is in the the current "throttled" state?
I'm using currently an algorithm during the loading phase and "charged" phase, but to read the state directly from a data point would be much easier ...

Thanks 1000x

**Comment by nobl** [comment]
Unfortunately I don't know if there is such an option.
Maybe checking with the folks over on the iobroker forums can help us identify one?

**Comment by nobl** [comment]
Wir könnten mal BMS_PARA.OPERATIONAL_MODE0 im Auge behalten.
Senec sagt "Konditionierungsmodus" und bei mir steht hier 3.

**Comment by dwm66** [comment]
Hab die auch schon im Blick, auch den USE_ROA_PARAMETER daneben. Da gabs ja die log Einträge mit COM70 und ROA50 als OpMode ... 
Ich tracke das jetzt mal in der Datenbank, und dann schau mer mal.

**Comment by nobl** [comment]
Da SENEC die Speicher inzwischen austauscht, wird das vermutlich nicht mehr wirklich benötigt.
